### PR TITLE
add pip log to archives

### PR DESCRIPTION
**Pull request by estute**
@jzoldak @benpatterson

### CODE REVIEW

File: platform/jobs/edxPlatformAccessibilityMaster.groovy [hunk 1]
```diff
@@ -130,7 +130,9 @@ secretMap.each { jobConfigs ->
            predefinedPropsMap.put('TARGET_URL', JENKINS_PUBLIC_BASE_URL +
                                   'view/accessibility/job/' + jobConfig['jobName'] + '/${BUILD_NUMBER}/')
            downstreamParameterized JENKINS_PUBLIC_GITHUB_STATUS_PENDING.call(predefinedPropsMap)
-           shell("cd ${jobConfig['repoName']}; RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh")
+           shell("cd ${jobConfig['repoName']}; " +
+                 'RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh; ' +
+                 "pip freeze > \${WORKSPACE}/${jobConfig['repoName']}/test_root/log/pip_freeze.log")
```
Comment (jzoldak) [inline platform/jobs/edxPlatformAccessibilityMaster.groovy]: If ./scripts/accessibility-tests.sh exits with an error code and then the pip freeze exits successfully, will the build pass or fail?

Comment (jzoldak) [inline platform/jobs/edxPlatformAccessibilityMaster.groovy]: @estute yeah I think that's a better approach, to create the log as part of a paver command.

Comment (estute) [inline platform/jobs/edxPlatformAccessibilityMaster.groovy]: @jzoldak i think we can close this one with https://github.com/edx/edx-platform/pull/13869/

**Comment by feanil** [comment]
Closing due to age and inactivity.

**Pull request closed without merging**